### DISCUSSION TITLE
[Question] Obfuscator messing up the paths when using pyinstaller.

### DISCUSSION THREAD

**Discussion by RoddyBenz** [dashingsoft/pyarmor#1103] [Q&A]
pyarmor gen --enable-jit --enable-themida --enable-rft --pack --recursive C:\Users\roddy\Downloads\tang

Pyarmor obfuscates the code but when using pyinstaller to pack the obfuscated code into exe:

pyinstaller --onefile tang.py

Traceback (most recent call last):
  File "<frozen __main__>", line 3, in <module>
  File "<frozen Tang.main>", line 1, in <module>
ImportError: cannot import name 'config' from 'helperfiles' (unknown location)
[36420] Failed to execute script 'main' due to unhandled exception!

packing the exe in older versions worked and I never saw issue with this before.

**Comment by RoddyBenz** [comment]
@jondy is there a way to fix this?

**Comment by tarekmabdallah91** [comment]
@jondy please your advice to solve this issue

**Comment by jondy** [comment]
Please read the documentation first.
